crates/ruff/src/rules/flake8_pyi/rules/long_numeric_literals_in_stub.rs

```diff
@@ -0,0 +1,30 @@
+use ruff_diagnostics::{Diagnostic, Violation};
+use ruff_text_size::{TextRange, TextSize};
+
+use ruff_macros::{derive_message_formats, violation};
+
+use crate::checkers::ast::Checker;
+
+#[violation]
+pub struct LongNumericLiteralsInStub;
+
+/// ## What it does
+/// Checks for numeric literals longer than 10 characters
+///
+/// ## Why is this bad?
+/// Long hardcoded numeric values are unlikely to be useful for users. Consider replacing them with ellipses.
```

Comment:
Can we add an example on how using ellipses looks? I wouldn't know what to do from reading the message itself.

Reply:
@MichaReiser added an example and updated the comment - lmk if looks good. thanks!